rviz_default_plugins/src/rviz_default_plugins/displays/wrench/wrench_display.cpp

```diff
@@ -127,11 +133,24 @@ bool validateFloats(const geometry_msgs::msg::WrenchStamped & msg)
 
 void WrenchDisplay::processMessage(geometry_msgs::msg::WrenchStamped::ConstSharedPtr msg)
 {
-  if (!validateFloats(*msg)) {
-    setStatus(
-      rviz_common::properties::StatusProperty::Error, "Topic",
-      "Message contained invalid floating point values (nans or infs)");
-    return;
+  auto custom_msg = std::make_shared<geometry_msgs::msg::WrenchStamped>();
```

Comment:
Maybe a better name. Moreover, this initialization should be moved lower only if `accept_nan` is used or maybe even better in the class, so we prevent initialization at each run.

```suggestion
  auto adjusted_msg = std::make_shared<geometry_msgs::msg::WrenchStamped>();
```